src/main/resources/static/js/createSurvey.js

```diff
@@ -10,6 +10,20 @@ textInputForm =`
      <textarea id="textQuestionPrompt" name="questionPrompt" rows="2" cols="80"></textarea><br>
      <button type="button" id="addTextQuestion">Add Question</button>
     </div>`
+	
+//Input HTML for creating a Range based question
+rangeInputForm =`
+    <br>
+    <div id="questionInputBlock">
+     <label for="rangeQuestionPrompt">Question:</label><br>
+     <textarea id="rangeQuestionPrompt" name="questionPrompt" rows="2" cols="80"></textarea><br>
+	 <label for="rangeQuestionMin">Range Minimum:</label><br>
+	 <textarea id="rangeQuestionMin" name="questionMin" rows="1" cols="10"></textarea><br>
```

Comment:
You could use <input type="number"> here instead of a <textarea> to prevent strings from being entered

Reply:
Thanks, I'll do that.  I have basically no experience with JS, so I was mostly cribbing off what was already written